dev/forbidden_from_release_tests/bin/main.dart

```diff
@@ -0,0 +1,268 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:file/file.dart';
+import 'package:file/local.dart';
+import 'package:path/path.dart' as path;
+import 'package:process/process.dart';
+
+const ProcessManager processManager = LocalProcessManager();
+const FileSystem fs = LocalFileSystem();
+
+Future<void> main(List<String> args) async {
+  final Options options = Options.fromArgs(args);
+
+  final Directory tempDirectory = fs.systemTempDirectory.createTempSync('forbidden_imports');
+  print('Using $tempDirectory for temporary files.');
+
+  await pubGet(options.flutter, options.target.parent);
+  final File dill = await compileKernel(
+    tempDirectory: tempDirectory,
+    dart: options.dart,
+    target: options.target,
+    feSnapshot: options.feSnapshot,
+    sdkProduct: options.sdkProduct,
+    packageConfig: options.packageConfig,
+    packageName: options.packageName,
+  );
+
+  final File v8SnapshotInfo = await compileAOTSnapshot(
+    tempDirectory: tempDirectory,
+    genSnapshot: options.genSnapshot,
+    dill: dill,
+  );
+
+  final String json = v8SnapshotInfo.readAsStringSync();
+  final List<String> foundForbiddenTypes = <String>[];
+  for (final String forbiddenType in options.forbiddenTypes) {
+    if (json.contains(forbiddenType)) {
+      foundForbiddenTypes.add(forbiddenType);
+    }
+  }
+  if (foundForbiddenTypes.isNotEmpty) {
+    print('The output contained the following forbidden types:');
+    print(foundForbiddenTypes);
+    exit(-1);
+  }
+
+  tempDirectory.deleteSync(recursive: true);
+}
+
+Future<void> pubGet(File flutter, Directory target) async {
+  final List<String> command = <String>[flutter.path, 'pub', 'get'];
+  print('Running command: ${command.join(' ')} in ${target.path}');
+  final ProcessResult pubGetResult = await processManager.run(
```

Comment:
Should we maybe stream the outputs to the console? (if this command hangs we will never see anything in the logs where it was hanging I think).

Reply:
(same for the other run commands)

Reply:
I'll change these.